### ISSUE TITLE
No fields in node.data.target when trying to render embedded entry (react)

### ISSUE THREAD

**Issue by ansh** [contentful/rich-text#147]
The README.md shows the way to render rich text as the following:

```
const options = {
  renderNode: {
    [BLOCKS.EMBEDDED_ENTRY]: (node) => {
      console.log(node)
      const { title, description } = node.data.target.fields;
      return <CustomComponent title={title} description={description} />
    }
  }
};
```

My implementation is basically identical but when I `console.log(node)` it gives the following result:

![Image of Code](https://i.imgur.com/XlX4dAd.png)
Source: https://imgur.com/XlX4dAd

Clearly there is no `fields` in `node.data.target`

I am super confused because the same code was working a few days ago but now its bust.

**Comment by LeunensMichiel** [comment]
I fixed it by doing gatsby clean (or cleaning your react cache if not using gatsby), by following this! Hope this helps!

#144

**Comment by ansh** [comment]
That is so odd because it was failing on my CI/CD (netlify and Gatsby Cloud) as well. That is so weird. 

Fixed issue by running `gatsby clean`

Closed, duplicate to #144
